conf/modules.config

```diff
@@ -152,55 +124,55 @@ process {
 
     withName: 'PARSE_IRMSD'{
         ext.prefix = { "${meta.id}_${meta.tree}-args-${meta.argstree_clean}_${meta.aligner}-args-${meta.args_aligner_clean}_irmsd" }
-        publishDir = [
```

Comment:
No! I remove them because they are intermediate files and the merged files contains all the information. I publish only the intermediate results that contain more information than the final merged file (like tcs, irmsd etx where we have information for each pair of sequences andin the final file we only put the avg)

Reply:
Fair enough, is there scenarios in which the user might want to have these files instead of the consolidated ones for processing them or this is something unlikely

Reply:
The ones I do not save are one-liner csvs that are then merged into the final csv (with all the other families) so you still have all the infos in the same exact format you just have everything together which is way more handy.